doc/lua_api.md

```diff
@@ -8234,32 +8234,38 @@ It can be created via `Settings(filename)`.
 ### Methods
 
 * `get(key)`: returns a value
+    * For the main settings object (`minetest.settings`), `get()` and other
+      getter functions fall back to default values for engine-defined settings.
```

Comment:
I think this should be reworded to better explain that the engine has internal defaults for its own settings but settingtypes.txt is not read.

also maybe move it before the `### Methods`

Reply:
Please suggest a better wording.

Reply:
```
Some settings of the  `minetest.settings` instance have internal defaults, which act as a fallback
when reading a setting that is not yet present in `minetest.conf`.
Note: These defaults *do yet not* originate from `builtin/settingtypes.txt` but C++ code.
```

Reply:
<pre>
Engine settings on the `minetest.settings` object have internal defaults that will be returned if unset.
In other cases the value will be empty. The engine does *not* (yet) read `settingtypes.txt` for this purpose.
</pre>

Reply:
Okay, what do you think about the new version?